doc/source/programs/gdal_vector_geom_op_explode_collections.rst

```diff
@@ -0,0 +1,80 @@
+.. _gdal_vector_geom_op_explode_collections_subsubcommand:
+
+================================================================================
+"gdal vector geom-op explode-collections" sub-sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Explode geometries of type collection of a vector dataset.
+
+.. Index:: gdal vector geom-op explode-collections
```

Comment:
Please don't make a change _only_ based on my comment, but I don't think "`-op`" is conveying much here. `gdal vector geom explode-collections` captures the same and is easier to remember, IMO.

Reply:
indeed ``geom`` should do the job

Reply:
renamed to geom